src/specific/display.c

```diff
@@ -10,10 +10,66 @@
 
 #include <stdlib.h>
 
+// The screen resolution is controlled by two variables are indices
+// within an array of predefined screen resolutions.
+// Actual screen resolution, sometimes different from the game
+// resolution (during FMVs, main menu sequence, static pictures etc.)
```

Comment:
this comment block could be written a lot better.

Reply:
so this was fixed up later, the fix-up could still do with some work on the second part.

Reply:
I'm open to suggestions.